dashboard/test/controllers/api/v1/teacher_scores_controller_test.rb

```diff
@@ -147,7 +147,7 @@ class Api::V1::TeacherScoresControllerTest < ActionDispatch::IntegrationTest
 
     post '/dashboardapi/v1/teacher_scores', params: {section_id: section.id, stage_scores: [{stage_id: lesson.id, score: score}]}
 
-    assert_queries 13 do
+    assert_queries 14 do
```

Comment:
I would love for someone to help me confirm that this update is ok but I think its because of the changes to the associations to the through relationships.

Reply:
this seems fine -- I wouldn't want to block a large refactor like this on an added query to a non-critical codepath.